packages/template-retail-react-app/app/pages/product-list/partials/refinements.jsx

```diff
@@ -139,7 +157,9 @@ const Refinements = ({filters, toggleFilter, selectedFilters, isLoading}) => {
 }
 
 Refinements.propTypes = {
+    itemsBefore: PropTypes.arrayOf(PropTypes.elementType),
```

Comment:
According to this [StackOverflow comment](https://stackoverflow.com/questions/42326084/what-is-the-correct-proptypes-for-component-or-element#comment116200253_55228237), what we want here is `element`, which is the element _instance_ (e.g. `<CategoryLinks key="itemsBefore" category={category} />`). `elementType` would be the type/definition (e.g. `CategoryLinks`).

```suggestion
    itemsBefore: PropTypes.arrayOf(PropTypes.element),
```